src/robot.coffee

```diff
@@ -393,26 +407,6 @@ class Robot
         scriptDocumentation.commands.push cleanedLine
         @commands.push cleanedLine
 
-  # Public: A helper send function which delegates to the adapter's send
-  # function.
-  #
-  # user    - A User instance.
-  # strings - One or more Strings for each message to send.
-  #
-  # Returns nothing.
-  send: (user, strings...) ->
-    @adapter.send user, strings...
```

Comment:
These methods violate the API contract of the adapters, sending a user instead of an envelope that wraps a user. I initially thought about using them instead of a response for `prereceive` middleware, but I ended up wanting to use a response so I could centralize handling `prereply` middleware.

They seem broken, though, so I went ahead and removed them. Tell me if I'm missing something?